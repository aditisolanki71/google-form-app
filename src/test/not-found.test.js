import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../components/not-found'

test('Snapshot of 404 Not found Component', () => {
   const component = renderer.create(
     <NotFound/>,
   );
   let tree = component.toJSON();
   expect(tree).toMatchSnapshot();

 });