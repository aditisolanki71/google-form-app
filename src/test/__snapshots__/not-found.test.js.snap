// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of 404 Not found Component 1`] = `
<div
  className="not-found"
>
  <h1>
    Page Not Found
  </h1>
</div>
`;
